twake/backend/node/src/services/notifications/services/engine/index.ts

```diff
@@ -28,6 +29,7 @@ export class NotificationEngine implements Initializable {
     gr.platformServices.messageQueue.processor.addHandler(
       new PushNotificationToUsersMessageProcessor(),
     );
+    gr.platformServices.messageQueue.processor.addHandler(new PushReactionNotification());
```

Comment:
Ok so the processor is initiated here so you can remove the TODO comment on my other message